src/epoll_shim_ctx.h

```diff
@@ -61,8 +61,10 @@ typedef struct {
 
 extern EpollShimCtx epoll_shim_ctx;
 
-FDContextMapNode *epoll_shim_ctx_create_node(EpollShimCtx *epoll_shim_ctx,
-    errno_t *ec);
+FDContextMapNode *epoll_shim_ctx_create_node(
```

Comment:
Not sure if there is any kind of ABI stability guarantee, but I added the new version just in case.

Reply:
For `epoll_shim_ctx_create_node` there aren't any ABI stability issues, you can just modify it.

Only the symbols annotated with `EPOLL_SHIM_EXPORT` are exported and shouldn't change.

Reply:
Okay good to know. Should I simplify this pull  request and just change the signature?

Reply:
Sure, please go ahead!